backends/p4tools/modules/testgen/README.md

```diff
@@ -26,44 +27,46 @@ These are the current usage flags:
+-D arg=value                           Defines a preprocessor symbol
+-U arg                                 Undefines a preprocessor symbol
+-E                                     Preprocess only. Prints preprocessed program on stdout.
+--nocpp                                Skips the preprocessor; assumes the input file is already preprocessed.
+--std {p4-14|p4-16}                    Specifies source language version.
+-T loglevel                            Adjusts logging level per file.
+--target target                        Specifies the device targeted by the program.
+--arch arch                            Specifies the architecture targeted by the program.
+--top4 pass1[,pass2]                   Dump the P4 representation after
+                                       passes whose name contains one of `passX' substrings.
+                                       When '-v' is used this will include the compiler IR.
+--dump folder                          Folder where P4 programs are dumped.
+-v                                     Increase verbosity level (can be repeated)
+--strict                               Fail on unimplemented features instead of trying the next branch.
+--max-tests maxTests                   Sets the maximum number of tests to be generated [default: 1]. Setting the value to 0 will generate tests until no more paths can be found.
+--stop-metric stopMetric               Stops generating tests when a particular metric is satisifed. Currently supported options are:
+                                       "MAX_STATEMENT_COVERAGE".
+--packet-size-range packetSizeRange    Specify the possible range of the input packet size in bits. The format is [min]:[max]. The default values are "0:72000". The maximum is set to jumbo frame size (9000 bytes).
+--out-dir outputDir                    The output directory for the generated tests. The directory will be created, if it does not exist.
+--test-backend testBackend             Select the test back end. P4Testgen will produce tests that correspond to the input format of this test back end.
+--input-branches selectedBranches      List of the selected branches which should be chosen for selection.
+--track-branches                       Track the branches that are chosen in the symbolic executor. This can be used for deterministic replay.
+--with-output-packet                   Produced tests must have an output packet.
+--path-selection pathSelectionPolicy   Selects a specific path selection strategy for test generation. Options are: DEPTH_FIRST, RANDOM_BACKTRACK, GREEDY_STATEMENT_SEARCH, and RANDOM_STATEMENT_SEARCH. Defaults to DEPTH_FIRST.
+--track-coverage coverageItem          Specifies, which IR nodes to track for coverage in the targeted P4 program. Multiple options are possible: Currently supported: STATEMENTS, TABLE_ENTRIES Defaults to no coverage.
+--saddle-point saddlePoint             Threshold to invoke multiPop on RANDOM_ACCESS_MAX_COVERAGE.
```

Comment:
Do we still support this?

Reply:
Technically yes, the name is now RANDOM_STATEMENT_SEARCH. I will update it. We can also drop it since I do not know how functional it is.